src/ui-kit/molecules/MovieList/MovieList.tsx

```diff
@@ -1,19 +1,31 @@
-import React from "react"
-import { movies } from "assets/movies";
+import * as React from "react"
 import { MovieElement } from "../MovieElement";
 import "./MovieList.scss"
 
-export const MovieList = () => { 
+type Props = {
+    deleteAction?: () => {};
+    editAction?: () => {};
+    movies?: Array<{}>;
+}
+
+export const MovieList: React.FC<Props> = (props: Props) => {
+    const { deleteAction, editAction, movies } = props;
+    
     return (
         <section className="listWrapper">
-            {movies.map((movie) => (
+            {movies.map((movie:any) => {
```

Comment:
let's add a separate type for a movie in the types folder
```suggestion
            ...
            {movies.map((movie: Movie) => {
```

types.ts
```typescript
type Movie = {
  id: string;
  ...
}
```